environments/generate_data.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import numpy as np
+from tqdm import tqdm
+
+from pendulum import Pendulum
+from spring import Spring
+
+# PARAMETERS
+
+dataset_root = '../datasets'
+dataset_name = "pendulum_data"
+num_train_samples = 1000
+num_test_samples = 100
+number_of_frames = 20
+delta_time = 0.125
+img_size = 32
+noise_std = 0.
+radius_bound = (.5, 1.5)
+color = False
+world_size = 1.5
+environment = Pendulum(mass=.5, length=1, g=3)
```

Comment:
On a second thought these parameters should definitely be taken from the yaml file? We risk generating the dataset with the wrong parameters. In fact, `number_of_frames` should be 30, not 20.

Reply:
of course